polynote-frontend/polynote/ui/component/notebook/cell.ts

```diff
@@ -609,6 +610,34 @@ abstract class Cell extends Disposable {
             .when("RunToCursor", () => {
                 this.dispatcher.runToActiveCell()
             })
+            .when("MoveAbove", () => {
+                this.notebookState.moveCell("above");
+                return ["stopPropagation", "preventDefault"]
+            })
+            .when("MoveBelow", () => {
+                this.notebookState.moveCell("below");
+                return ["stopPropagation", "preventDefault"]
+            })     
+            .when("Merge", () => {
+                const nextCellId = this.notebookState.getNextCellId(this.id);
+                if (nextCellId) {
+                    const nextCellContent = this.notebookState.state.cells[nextCellId].content;
+                    this.appendText(nextCellContent);
+                    this.notebookState.deleteCell(nextCellId);
+                }
+                return ["stopPropagation", "preventDefault"]
+            })        
+            .when("Split", () => {
```

Comment:
This is definitely safer than Merging, but I'm a little on the fence about it. How easy is this to undo if hit accidentally?

Thoughts @jeremyrsmith ?